package.json

```diff
@@ -17,40 +17,40 @@
   },
   "private": true,
   "peerDependencies": {
-    "@angular/animations": "~15.2.10",
-    "@angular/cdk": "~15.2.9",
-    "@angular/common": "~15.2.10",
-    "@angular/compiler": "~15.2.10",
-    "@angular/core": "~15.2.10",
-    "@angular/forms": "~15.2.10",
-    "@angular/material": "~15.2.9",
-    "@angular/platform-browser": "~15.2.10",
-    "@angular/platform-browser-dynamic": "~15.2.10",
-    "@angular/router": "~15.2.10",
+    "@angular/animations": "~18.2.13",
+    "@angular/cdk": "~18.2.14",
+    "@angular/common": "~18.2.13",
+    "@angular/compiler": "~18.2.13",
+    "@angular/core": "~18.2.13",
+    "@angular/forms": "~18.2.13",
+    "@angular/material": "~18.2.14",
+    "@angular/platform-browser": "~18.2.13",
+    "@angular/platform-browser-dynamic": "~18.2.13",
+    "@angular/router": "~18.2.13",
     "rxjs": "~7.4.0",
-    "zone.js": "^0.11.4"
+    "zone.js": "^0.14.10"
   },
   "dependencies": {
-    "@ngx-translate/core": "14.0.0",
-    "arlas-wui-toolkit": "27.0.0-rc.1"
+    "arlas-wui-toolkit": "27.0.0-beta.7"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^15.2.11",
-    "@angular-eslint/builder": "15.2.1",
-    "@angular-eslint/eslint-plugin": "15.2.1",
-    "@angular-eslint/eslint-plugin-template": "15.2.1",
-    "@angular-eslint/schematics": "15.2.1",
-    "@angular-eslint/template-parser": "15.2.1",
-    "@angular/cli": "^15.2.11",
-    "@angular/compiler-cli": "^15.2.10",
-    "@angular/language-service": "^15.2.10",
+    "@angular-devkit/build-angular": "^18.2.12",
+    "@angular-eslint/builder": "^18.2.12",
+    "@angular-eslint/eslint-plugin": "^18.2.12",
+    "@angular-eslint/eslint-plugin-template": "^18.2.12",
+    "@angular-eslint/schematics": "^18.2.12",
+    "@angular-eslint/template-parser": "^18.2.12",
+    "@angular/cli": "^18.2.12",
+    "@angular/compiler-cli": "^18.2.13",
+    "@angular/language-service": "^18.2.13",
     "@biesbjerg/ngx-translate-extract-marker": "1.0.0",
+    "@stylistic/eslint-plugin": "^2.12.1",
+    "@stylistic/eslint-plugin-ts": "^2.12.1",
     "@types/jasmine": "~3.10.0",
     "@types/jasminewd2": "~2.0.2",
     "@types/node": "^18.19.0",
-    "@typescript-eslint/eslint-plugin": "^5.43.0",
-    "@typescript-eslint/parser": "^5.43.0",
-    "@vendure/ngx-translate-extract": "8.3.0",
+    "@typescript-eslint/eslint-plugin": "^8.18.0",
```

Comment:
should we remove them as they're replace by @stylistic/eslint-plugin ?

Reply:
this is still needed ?

Reply:
This is used in the .eslintrc.json file, in complement to the new `@stylistic` rules